pom.xml

```diff
@@ -24,66 +31,26 @@
         <url>http://github.com/elasticsearch/elasticsearch</url>
     </scm>
 
-    <parent>
-        <groupId>org.sonatype.oss</groupId>
-        <artifactId>oss-parent</artifactId>
-        <version>7</version>
-    </parent>
-
     <properties>
-        <lucene.version>5.1.0</lucene.version>
-        <lucene.maven.version>5.1.0-snapshot-1657571</lucene.maven.version>
-        <tests.jvms>auto</tests.jvms>
-        <tests.shuffle>true</tests.shuffle>
-        <tests.output>onerror</tests.output>
-        <tests.client.ratio></tests.client.ratio>
-        <tests.bwc.path>${project.basedir}/backwards</tests.bwc.path>
-        <tests.locale>random</tests.locale>
-        <tests.timezone>random</tests.timezone>
-        <es.logger.level>INFO</es.logger.level>
-        <tests.heap.size>512m</tests.heap.size>
-        <tests.heapdump.path>${basedir}/logs/</tests.heapdump.path>
-        <tests.topn>5</tests.topn>
-        <execution.hint.file>.local-${project.version}-execution-hints.log</execution.hint.file>
+        <!-- We inherit from common project properties but we can still force our settings -->
     </properties>
 
-    <repositories>
-        <repository>
-            <id>codehaus-snapshots</id>
-            <name>Codehaus Snapshots</name>
-            <url>http://repository.codehaus.org/</url>
-        </repository>
-        <repository>
-            <id>lucene-snapshots</id>
-            <name>Lucene Snapshots</name>
-            <url>https://download.elasticsearch.org/lucenesnapshots/1657571</url>
-        </repository>
-    </repositories>
-
     <dependencies>
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-all</artifactId>
-            <version>1.3</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>com.carrotsearch.randomizedtesting</groupId>
             <artifactId>randomizedtesting-runner</artifactId>
-            <version>2.1.11</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.lucene</groupId>
             <artifactId>lucene-test-framework</artifactId>
-            <version>${lucene.maven.version}</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.3.5</version>
-            <scope>test</scope>
```

Comment:
This is not test scope in the parent project. We should either make it test scope in the parent pom or explicitly set test scope here